src/cdqr.py

```diff
@@ -17,6 +17,11 @@
 except:
     compression = zipfile.ZIP_STORED
 
+try:
```

Comment:
Can you explain a bit why this was added? It seems it would conflict with those places where the input() function is being used. Such as https://github.com/orlikoski/CDQR/blob/master/src/cdqr.py#L654

Reply:
https://stackoverflow.com/questions/33946678/python-how-to-hack-input

Python 3 does not have a raw_input and it will just except and pass, if anyone tries to run it in python2.7 it will disallow the issue expressed in the stackoverflow above. Until python 2 is officially EOL it is the safer thing to do even for projects like this that are explicitly python3 in case someone accidentally tries to run it in python2.

Reply:
My understanding is that this works as is in both Py2.x and Py3.x as Py3 has a built in alias from raw_input() to input().

That said, CDQR will break for many other reasons with Py2.x and this project has always been a Python 3 build. Now, a check for python 2 vs 3 and stopping it with an error message instead of this would be a nice addition

Reply:
I can do that! I'll edit this tonight!

Reply:
I just edited it, check the verbiage if you would like. As a side note I used to thing the same thing but there is no raw_input() in python 3, Just verified. That try catch with input/raw_input was a trick I learned from Mark Baggett.  If there are any other additions you have been thinking of adding let me know.

Reply:
Nice! Slightly changed the wording on the error message and looks ready to go